src/util/allIcons.ts

```diff
@@ -118,4 +118,5 @@ export const ALL_ICONS = [
   'view-headline',
   'view-module',
   'warning',
+  'yield',
```

Comment:
Called the `error` icon with the dark exclamation mark `yield` to match tokens

Reply:
Which token is that? I was wondering if we should name it something with "error" so it's easier for people to find, like `error-inverted` or something.

Reply:
I didn't want to confuse `border-utility-error` and `text-utility-error` so in hindsight `revise` might have been better but the variant naming is I think synced with `yield` (hopefully).

Do agree to call it `error-inverted` tho